build/nse/universal-cnf/Makefile

```diff
@@ -5,5 +5,6 @@ AUX_CONTAINERS =
 PODS =
 NETWORK_SERVICES =
 CHECK =
+VPP_AGENT=ligato/vpp-agent:v3.2.0
```

Comment:
The root nsm-nse/Makefile sets this already.  This shouldn't need to be set here.

Reply:
@tiswanso Actually there is a Makefile bug that makes the value of VPP_AGENT be overwritten by the kiknos NSE where it is set to kiknos agent.. and this value is then passed most likely via the for each loops and includes in makefiles.. and causing the the universal-cnf NSE image to use wrong base image (from kiknos).. which is not obvious from user perspective at all..